tabular-data-analysis/docker-compose-lit.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+
+services:
+  localai:
+    image: localai/localai:v2.6.1-ffmpeg-core
+    command: --models-path /models --context-size 4096 --threads 2
+    volumes:
+      - $PWD/models:/models
+    ports:
+      - "10080:8080"
+  tabular-data-analysis:
+    image: sunysb/tabular-data-analysis:lit-1.0
```

Comment:
The user should be able to build this image by themselves and use their own image, right? We should only depend on publicly available and well-established images.

Reply:
Add run.sh script to build docker image on the fly.